src/GovUk.Frontend.AspNetCore/TagHelpers/CharacterCountTagHelper.cs

```diff
@@ -194,33 +194,22 @@ public decimal? Threshold
 
     private protected override TagBuilder CreateTagBuilder(bool haveError, IHtmlContent content, TagHelperOutput tagHelperOutput)
     {
-        if (MaxLength.HasValue && MaxWords.HasValue)
-        {
-            throw new InvalidOperationException($"Only one of the '{MaxLengthAttributeName}' or '{MaxWordsLengthAttributeName}' attributes can be specified.");
-        }
-
-        // N.B. We specifically pass FormGroupAttributes here and not tagHelperOutput.Attributes since GenerateCharacterCount() doesn't return a form group
-        var formGroup = Generator.GenerateFormGroup(
-            haveError,
-            content,
-            FormGroupAttributes.ToAttributeDictionary());
-
         var resolvedId = ResolveIdPrefix();
-
         return Generator.GenerateCharacterCount(
+            haveError,
             resolvedId,
             MaxLength,
             MaxWords,
             Threshold,
-            formGroup,
+            content,
             CountMessageAttributes.ToAttributeDictionary(),
-            null,
-            null,
-            null,
-            null,
-            null,
-            null,
-            null);
+            textAreaDescriptionText: null,
+            charactersUnderLimitText: null,
+            charactersAtLimitText: null,
+            charactersOverLimitText: null,
+            wordsUnderLimitText: null,
+            wordsAtLimitText: null,
+            wordsOverLimitText: null);
     }
```

Comment:
- New argument of `haveError` to make it available in the generator (I don't recall why it is required, but did lots of back-and-forth adding it/removing it/adding it/removing it etc.)
- Renamed argument `formGroup` -> `content` to recognise the `govuk-form-group` is now the root element and not a child
- Given the long run of `null`s, I've added named arguments as an improvement to prevent accidents with positional arguments being mis-positioned.
  - Should these even be null, as opposed to being populated from somewhere?